src/pkg/cli/client/byoc/aws/stream.go

```diff
@@ -138,7 +138,8 @@ func (bs *byocServerStream) parseEvents(events []ecs.LogEvent) (*defangv1.TailRe
 	if bs.etag != "" && bs.etag != response.Etag {
 		return nil, nil // TODO: filter these out using the AWS StartLiveTail API
 	}
-	if bs.service != "" && bs.service != response.Service {
+
+	if len(bs.services) > 0 && !pkg.Contains(bs.services, bs.response.GetService()) {
```

Comment:
any reason why this is `Contains` and not equality?

Reply:
the bs object now has a list of services rather than just one (populated by TailRequest.Services), the response object is only for one service so we are just checking that the service is in the list of services we are interested in.